implot_demo.cpp

```diff
@@ -1880,7 +1880,7 @@ void Demo_OffsetAndStride() {
         ImPlot::PushColormap(ImPlotColormap_Jet);
         char buff[16];
```

Comment:
Should be at least 20 bytes (8 bytes format text + 10 bytes for `%d`). Could be even 32; to align with buffer size used above.
```suggestion
        char buff[32];
```